src/Block_Energy.f90

```diff
@@ -31,6 +31,6 @@ module Block_Energy
 
 !   Some important constants
 !
-      real   :: lvp,rb,rho,evap_coeff=1.5e-9,pf=0.640,pi=3.14159       
+      real   :: lvp,rb,rho,evap_coeff=1.5e-9,pf=0.640,pi=3.14159       
```

Comment:
Declare the variables on individual lines and include a comment to indicate what they are.